src/main/java/com/commercetools/sync/commons/utils/CustomUpdateActionUtils.java

```diff
@@ -61,8 +61,10 @@
         @Nonnull final S newResource,
         @Nonnull final GenericCustomActionBuilder<T> customActionBuilder,
         @Nonnull final BaseSyncOptions syncOptions) {
-
-        return buildCustomUpdateActions(oldResource, newResource, customActionBuilder, null,
+        return buildCustomUpdateActions(
+            null,
```

Comment:
Is it possible to use overloading and create 2 different methods instead of passing nulls?

Reply:
620534df